src/editors/_shared/hooks/useDataTypes.js

```diff
@@ -1,78 +1,40 @@
+import { useMemo, useCallback } from "react";
 import { useTheme } from "@material-ui/styles";
 import DataTypeManager from "./DataTypes/DataTypeManager";
 
-const useDataTypes = () => {
+const useDataTypes = (options = {}) => {
+  const { onlyStrings } = options;
+
   // Hooks
   const theme = useTheme();
-  const dataTypeManager = new DataTypeManager({ theme });
-
-  //========================================================================================
-  /*                                                                                      *
-   *                                    Exposed Methods                                   *
-   *                                                                                      */
-  //========================================================================================
+  const dataTypeManager = useMemo(
+    () => new DataTypeManager({ theme, onlyStrings }),
```

Comment:
I think you didn't do this. But why the hell we need to pass the `theme` to a class constructor?

Reply:
Because it goes to some of the edit components